frontend/public/actions/features.ts

```diff
@@ -252,6 +254,19 @@ const detectConsoleLinks = (dispatch) =>
     },
   );
 
+const detectIndependentMode = (dispatch) => {
+  k8sGet(OCSServiceModel, 'ocs-storagecluster', 'openshift-storage')
```

Comment:
Since you want to react on creation/deletion of CR you should use `k8sWatch` instead of `k8sGet`

Also this `detectIndependentMode` function should be contributed via new extension point by ceph plugin

Reply:
ack